### DISCUSSION TITLE
create a permanent 'documentation' branch

### DISCUSSION THREAD

**Discussion by umlaeute** [pure-data/pddp#175] [Ideas]
similar to the `devel` branch, we should have a `documentation` branch in the *main* Pd repository [pure-data/pure-data](https://github.com/pure-data/pure-data).

- people can create PRs against this branch
- @porres should be able to push to that branch

#### thoughts
every now and again somebody comes up with a small documentation fix which they submit as a PR against the `master` branch.
having to tell them to create a new PR in a different repository is rather unfriendly (and while we can re-target a PR to a different branch in the same repository, we cannot re-target it to a different repository)

to be honest, I would like to restrict @porres to only be able to push to this documentation branch (so we don't get accidental pushes to e.g. `master`) (sorry bro, but I'm still not 100% convinced of your git foo :stuck_out_tongue_winking_eye: :ninja:  )

while porres' docs branch has traditionally been called `Documentation`, I would suggest to use `documentation` (all lower-case) for that branch - virtually all branches in the main repository use a lowercase spelling (with exceptions for abbreviations like `API`)

**Comment by ben-wes** [comment]
sounds good - although for me all options are fine (besides my definite lack of actual git foo). what was a little confusing about my previous PR was that i tried to follow the instructions on https://github.com/pure-data/pddp, which state that `Documentation changes regarding contents of https://github.com/pure-data/pure-data (manual/help files) should be made via a PR to that repository.`

i just mentioned this in the updated PR as well: https://github.com/porres/pure-data/pull/34#issuecomment-1828385350

**Comment by porres** [comment]
sounds great

**Comment by umlaeute** [comment]
- [x] created [`documentation`](https://github.com/pure-data/pure-data/tree/documentation) branch
- [x] migrated the old "Documentation updates" PR to https://github.com/pure-data/pure-data/pull/2147
- [x] created a @pure-data/docs team for documentation writers
- [x] updated branch protection rules, so the @pure-data/docs team can push
- [x] invited @porres to the `docs` team
- [x] @porres accepted membership to the team

**Comment by porres** [comment]
done
